src/pages/Header/Header.jsx

```diff
@@ -28,18 +29,25 @@ const SearchInput = styled(Input)({
   width: '100% !important',
 });
 
-function Header() {
+function Header({ setTerm }) {
   const { authenticated, user, logout } = useAuth();
   const history = useHistory();
 
+  const onKeyPress = (event) => {
+    if (event.key === 'Enter' && event.target.value !== '') {
+      setTerm(event.target.value);
+    }
+  };
+
   return (
     <TopMenu fixed="top">
-      <Item header>Wizeline</Item>
+      <Item header onClick={() => history.push('/home')}><h3>Wizeline</h3></Item>
       <Search>
         <SearchInput
           className="icon search"
           icon="search"
           placeholder="Buscar..."
+          onKeyPress={(e) => onKeyPress(e)}
```

Comment:
there is no need to create a function that has `e` as params and calls another function that only has `e` as params, it can be changed to:

```suggestion
          onKeyPress={onKeyPress}
```